06-building-a-program-start-to-finish/reflecting-on-the-learning-experience/Output/README.md

```diff
@@ -0,0 +1,7 @@
+Output
+
+Link to Wiki Medium Post https://medium.com/@faith.omokaro/checkpoints-and-simulations-my-road-to-learning-38c3ccf07c09#.e27qrpifq
+
+Author
+
+Omokaro Faith
```

Comment:
Good read. 

It will help you to know that your ReadMe.md files are not properly formatted. [See link](https://guides.github.com/features/mastering-markdown/)